server/templates/transcribe.html

```diff
@@ -34,6 +34,10 @@
         </audio>
         <textarea id="readwrite-textbox" rows="4"></textarea>
 
+        <div class="button-container">
+            <button id="dark-mode-button" class="dark-mode-button">שינוי בהירות</button>
```

Comment:
This is in the middle of the work area - not a great place for this.
Also, a button is weird.

Can you:

1. Make this a switch (https://developer.android.com/develop/ui/views/components/togglebutton)
2. Move it to the upper right corner (same height as user login info)

Thanks :)

Reply:
1. i didn't want to import android CSS sheets, so i used classic CSS instead.
2. done